pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -169,10 +176,10 @@ func (r *Reconciler) updateStatus(ctx context.Context, desired *v1alpha1.PubSub)
 
 	if err == nil && becomesReady {
 		duration := time.Since(src.ObjectMeta.CreationTimestamp.Time)
-		r.Logger.Infof("PubSub %q became ready after %v", source.Name, duration)
+		logging.FromContext(ctx).Desugar().Info("PubSub became ready", zap.Any("after", duration))
```

Comment:
I might add an `Eventf` call here. The transition from unready to ready (and possibly vice versa) seems like it deserves a k8s event of its own since most of the time that's what users will be looking for.